lib/installed-package-view.coffee

```diff
@@ -53,12 +53,16 @@ class InstalledPackageView extends View
       @div outlet: 'sections'
 
   initialize: (@pack, @packageManager) ->
+    @activate()
     @populate()
     @handleButtonEvents()
     @updateFileButtons()
     @checkForUpdate()
     @subscribeToPackageManager()
 
+  activate: ->
+    @pack.activateConfig() if not atom.packages.isPackageActive(@pack.name)
```

Comment:
I think this should be done in a next tick (like it used to be) since activating the config for ~100 packages will probably take some time and at least the settings view will get a chance to render and "feel" like it is continuing to load while the configs are activating.